core/src/classes/codeConfig.ts

```diff
@@ -117,15 +118,16 @@ export function validateConfigObjectKeys(
 
 export function logModel(
   instance,
-  mode: "created" | "updated" | "deleted" | "validated"
+  mode: "created" | "updated" | "deleted" | "validated",
+  name?: string
 ) {
   let logLevel = "info";
   if (mode === "created") logLevel = "notice";
   if (mode === "deleted") logLevel = "warning";
 
   log(
     `[ config ] ${mode} ${instance.constructor.name} \`${
-      instance.key || instance.email || instance.name
+      name || instance.key || instance.email || instance.name
```

Comment:
I think I missed where did the new `name` property get used?  It looks like the new Profile config objects still have `id`?

```
    {
      id: "profile_john",
      class: "Profile",
      properties: {
        user_id: 20,
      },
    },
```

Reply:
profiles don't have a name, so logging it produced `undefined`:

```
notice: [ config ] created Profile `undefined` (pro_440aee4d-a936-46f2-a3ee-ca822b405d3c)
```

I'm kind of faking a name by passing the stringified properties:
```
notice: [ config ] created Profile `{"user_id":21}` (pro_440aee4d-a936-46f2-a3ee-ca822b405d3c)
```

Reply:
Ah! That's a good log message too.  Carry on then!